IdentityCore/src/oauth2/MSIDExternalSSOContext.m

```diff
@@ -41,6 +49,12 @@ - (MSIDWPJKeyPairWithCert *)wpjKeyPairWithCertWithContext:(id<MSIDRequestContext
             return nil;
         }
         
+        if (!self.loginManager.isDeviceRegistered || (self.loginManager.isDeviceRegistered && ![MSIDExternalSSOContext isPlatformSSORegisteredFlagSetInUserDefaults]))
+        {
+            MSID_LOG_WITH_CTX(MSIDLogLevelError, context, @"No valid PSSO registration found on device, returning early");
```

Comment:
Should we print those parameters (self.loginManager.isDeviceRegistered and MSIDExternalSSOContext isPlatformSSORegisteredFlagSetInUserDefaults]) in case it helps with debugging?

Reply:
added